test/test_estimates_api.py

```diff
@@ -183,6 +183,18 @@ def test_create_ethereum_estimate_transaction_value(self):
             estimate.data.mass_g, 1
         )  # not setting an exact value since this is changing daily
 
+    def test_create_hotel_estimate(self):
+        """Test case for create_hotel_estimate
```

Comment:
This test is failing. But it's not obvious to me why. I'm getting a keyword error on the `country_code` arg, but it doesn't seem like that should be causing a problem since it is a required argument. 

Here's a direct copy of the error message:
"Got an unexpected keyword argument 'country_code' to method create_hotel_estimate"

Anyone have an idea what the issue might be here? Is there something bespoke about the python generation process?

Reply:
@boconnell2 I can't check the issue right now, but my gut feeling is that this variable need to be allowed specifically for the Python SDK (one of multiple quirks). Check out this past PR: https://github.com/patch-technology/client-code-generation/pull/57/files#r847829571. I think if you do the same thing that might solve your issue.